lockfile/src/parsers/pypi.rs

```diff
@@ -17,7 +17,10 @@ pub fn parse(input: &str) -> Result<&str, Vec<Package>> {
     let mut pkgs = Vec::new();
 
     // Iterate over all non-empty lines.
-    for line in input.lines().filter(|line| !line.starts_with('#') && !line.trim().is_empty()) {
+    for line in
+        input.lines().filter(|line| !line.trim().starts_with('#') && !line.trim().is_empty())
```

Comment:
Would probably be more efficient to assign `line.trim()` to a temporary variable. Seems like the kind of thing that you don't necessarily want to repeat.

Reply:
I _think_ I was able to make the suggested change. Rust is still very much unnatural to me and so I welcome any idiomatic improvements.